node-graph/preprocessor/src/lib.rs

```diff
@@ -24,6 +24,36 @@ pub fn expand_network(network: &mut NodeNetwork, substitutions: &HashMap<ProtoNo
 	}
 }
 
+/// Referencing the TaggedValue::IndexOperationFilter will expand to a node that contains the evaluate_index_operation_filter
+pub fn evaluate_index_operation_filter(network: &mut NodeNetwork) {
```

Comment:
I don't see why we want to convert the input to a node that outputs a boolean, instead of just directly using the IndexOperationFilter value within the transform node or whatever node needs to use it

Reply:
Because `IndexOperationFilter` is a type that can be fed by a value input from the Properties panel to give it convenient index-based ranges, while more flexibility can be achieved by passing in any arbitrary field-style node input branch that evaluates to bool per instance.

Reply:
We already have a node that converts a range value to the boolean value.